test/unit_tests/services/eeprom_emulation.cpp

```diff
@@ -1184,7 +1185,7 @@ TEST_CASE("Migration from legacy format", "[eeprom]")
     TestEEPROM eeprom;
 
     // Load EEPROM extracted from a Photon into memory
-    loadEEPROMFromFile("eeprom_page1.bin", eeprom, PageBase1, PageSize1);
+    loadEEPROMFromFile(TEST_DIRECTORY "/eeprom_page1.bin", eeprom, PageBase1, PageSize1);
```

Comment:
I suggest we move these to a subdirectory in the test suite's source directory, e.g. `fixtures`.

Reply:
:+1: Agreed. Let's do that.